src/span_context.js

```diff
@@ -82,6 +89,14 @@ export default class SpanContext {
         this._baggage = baggage;
     }
 
+    set debugId(debugId: string): void {
+        this._debugId = debugId;
+    }
+
+    isDebugIDContainerOnly(): boolean {
+        return !this._traceId && this._debugId !== '';
```

Comment:
I would do `return !this._traceId && this._debugId;`, don't need to check against empty string